doc/user/content/manage/access-control/_index.md

```diff
@@ -1,119 +1,139 @@
 ---
-title: "Access control"
-description: "How to configure and manage access control in Materialize."
+title: "Access control (Role-based)"
+description: "How to configure and manage role-based access control (RBAC) in Materialize."
 disable_list: true
+aliases:
+  - /manage/access-control/rbac/
 menu:
   main:
     parent: manage
-    name: Access control
     identifier: 'access-control'
-    weight: 10
+    weight: 12
 ---
 
 {{< note >}}
 Configuring and managing access control in Materialize
-requires **administrator** privileges.
+requires **administrator** privileges; e.g., **Organization Admin** role.
 {{</ note >}}
 
-Access control in Materialize is configured at two levels: access to the
-[Materialize Console](https://console.materialize.com/) and access within the
-database. The privileges assigned on user invitation have implications at both
-levels, so we recommend carefully evaluating your access control needs ahead of
-expanding the number of users in your Materialize organization.
+<a name="role-based-access-control-rbac" ></a>
 
-## Account management
```

Comment:
This section is moved to the new Users/Service accounts section:
- Rendered: https://preview.materialize.com/materialize/33102/manage/users-service-accounts/
- In this PR: https://github.com/MaterializeInc/materialize/pull/33102/files#diff-226689f9ce45c3bf68c05bd21db2cd57f85e72e950fd2af225926b5030b364f8